be/src/olap/olap_common.h

```diff
@@ -189,20 +186,29 @@ struct Version {
     int64_t second;
 
     Version(int64_t first_, int64_t second_) : first(first_), second(second_) {}
-
     Version() : first(0), second(0) {}
 
+    friend std::ostream& operator<<(std::ostream& os, const Version& version);
+
     bool operator!=(const Version& rhs) const {
         return first != rhs.first || second != rhs.second;
     }
 
     bool operator==(const Version& rhs) const {
         return first == rhs.first && second == rhs.second;
     }
+
+    bool contains(const Version& other) const {
```

Comment:
first <= other.first && second >= other.second

Reply:
Done. Thank you for your careful review.